ui/css/_listOfLinks.scss

```diff
@@ -0,0 +1,21 @@
+
+.listOfLinks ul {
+  padding: 0 0 0 2rem;
+}
+
+.listOfLinks li {
+  padding:0;
+  font-size:1.3rem;
+}
+
+.listOfLinks h3{
+  margin:2rem 0 1rem 0;
+}
+
+.listOfLinks a:after{
+  content: ' ⮕';
```

Comment:
This doesn't appear as an arrow, but rather, just how it does here in the css; a `⮕`. Is the icon font missing?

Reply:
No, I'm using what you see there, the ⮕ character. CSS injection doesn't support anything other than inline chars (no enitity encoding or html) although we can use unicode. We can select a different character if you like.  In general I'm  on the fence about this design solution, since the definition of links on the internet is underlined and clickable text (and we already have both) I'm not sure we need another device, but I'm not opposed to adding some kind of visual "more" character if the client wants it.

Reply:
Argh, super annoying, it seems the right-arrow is a weird thing: https://stackoverflow.com/questions/4686518/cant-find-a-unicode-right-arrow

I could probably make it work on your system (and mine) but I'm not confident it will work everywhere, so as a workaround I'm going to put the (slightly different) FA right arrow in the HTML directly instead of CSS inject. Much less elegant but same visual effect and more reliable.